LICENSE

```diff
@@ -1,4 +1,5 @@
 Copyright (c) 2014-2015, Causality Ltd.
+Copyright (C) 2016-2017, The Pony Developers
```

Comment:
Hmm... I think I've usually seen the most recent copyright line as being the first line.

Any reason why you put it on the second line here?